src/main/java/bot/Storage.java

```diff
@@ -0,0 +1,82 @@
+package main.java.bot;
+
+import main.java.task.Task;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+
+/**
+ * A class that handles the storage of list data in a txt format.
+ * The data is stored in the local hard drive.
+ */
+public class Storage {
+
+    private static final String FILE_PATH = "data/duke.txt";
+
+    /**
+     * Returns a TaskList object de-serialised from the local data.
+     * If no local data exists, returns an empty TaskList object.
+     *
+     * Takes no parameters.
+     *
+     * @return A TaskList object that will contain all the tasks given to Duke by the user.
+     * @throws DukeException if there is an error loading the local data.
+     */
+    public static TaskList load() throws DukeException {

```

Comment:
Perhaps you could name this loadTasks to make it clear that a list of tasks is returned?